spec/factories/forms.rb

```diff
@@ -65,7 +65,8 @@ def build_item(item, form, parent, evaluator)
       form: form,
       group_name_en: item[:name],
       group_hint_en: item[:name],
-      repeatable: true
+      repeatable: true,
+      group_item_name_translations: item[:group_item_name_translations]
```

Comment:
Can you please do this one in the same style as the others, i.e. just do English, so `group_item_name_en: item[:item_name]` perhaps.

Reply:
ok